packages/nextjs/pages/batches/index.tsx

```diff
@@ -333,4 +284,75 @@ const Batches = () => {
   );
 };
 
+export const getStaticProps: GetStaticProps<PageProps> = async () => {
+  try {
+    const [batchesResponse, buildersResponse] = await Promise.all([
+      fetch(`${process.env.NEXT_PUBLIC_BG_BACKEND_API}/batches`),
+      fetch(`${process.env.NEXT_PUBLIC_BG_BACKEND_API}/builders`),
```

Comment:
I think this is too much. Fetching ALL the builders and iterating thru all of them is a bit crazy.

Let's improve the API endpoints on bgv3: the /batches endpoint, or even adding a new /batches/stats, where we get all the data we need. Since we have access to the database we can query all the builders with a non-null batch.

Reply:
Update: see https://github.com/scaffold-eth/buidlguidl-v3/pull/118#discussion_r1830777247

Let's fix it there so we can update here too!

Reply:
Updated with the new totalParticipants field from the endpoint 🙏